lib/Dialect/LLHD/Transforms/Deseq.cpp

```diff
@@ -1222,14 +1223,14 @@ ValueRange Deseq::specializeProcess(FixedValues fixedValues) {
       auto [oldBlock, newBlock] = worklist.pop_back_val();
       builder.setInsertionPointToEnd(newBlock);
       for (auto &oldOp : *oldBlock) {
-        // Convert `llhd.wait` into `scf.yield`.
+        // Convert `llhd.wait` into `llhd.halt`.
```

Comment:
nit: does this comment correct?

Reply:
No you're right, this should be `llhd.yield`. Thanks! 🙌